src/pyconnect/core.py

```diff
@@ -26,6 +28,34 @@ def message_repr(msg: Message) -> str:
     )
 
 
+def hash_sensitive_values(
+    config: Dict[str, Any], algorithm: str = "sha256", iterations: int = 100000, log_params: bool = True
+) -> Dict[str, Any]:
+    """
+    This function takes a kakfa configuration dictionary and hashes all present sensitive values (i.e. any keys from
+    "ssl.key.password", "ssl.keystore.password", "sasl.password", "ssl.key.pem", "ssl_key" which are in the dictionary).
+    By default the hashing parameters are logged. If you do not want this behavior, set `log_params` to False.
+    Note: The original dictionary is not modified.
+
+    :param config: Kafka config dictionary.
+    :param algorithm: Hash algorithm.
+    :param iterations: Number of times to run the hashing algorithm.
+    :param log_params: Should the hashing parameters be logged?
```

Comment:
```suggestion
    :param log_params: Should the hashing parameters be logged? Set to `False` if you don't need to be able to check the secret value. Since the salt and therefore the secret's hash is completely random, not providing the parameters makes the output basically equivalent to masking the secret with a static string like "***".
```
Thinking about it, if there's no use for the computed hash without parameters, there is also no need to compute it in the first place. Maybe there could be two functions: `hash_sensitive_values` and `hide_sensitive_values` and which one is used depends on some configuration setting.

Reply:
I changed the flag to `hash_passwords`. If it is false then the hashes are not computed and the secret value is replaced by "****"